cypress/integration/workflow-actions.spec.js

```diff
@@ -96,47 +96,11 @@ describe('Workflow Actions', () => {
     const confirmBtn =
       '[data-testid="reset-confirmation-modal"] [data-testid="confirm-modal-button"]';
 
-    it('to the first workflow task if the workflow is running, and the action is enabled', () => {
-      cy.get('#workflow-actions-menu-button').click();
-      cy.get('#workflow-actions-menu >> [data-testid="reset-button"]').click();
-      cy.get('[data-testid="workflow-reset-type-select"]').click();
-      cy.get('#reset-type-select-options > li[role="option"]').first().click();
-      cy.get('#reset-reason').type('test');
-      cy.get(confirmBtn).click();
-
-      cy.wait('@reset-workflow-api');
-      cy.waitForWorkflowAPIs();
-
-      cy.get('[data-testid="workflow-reset-alert"]').should('be.visible');
-      cy.get(confirmBtn).should('not.be.visible');
-    });
-
-    it('to the last workflow task if the workflow is running, and the action is enabled', () => {
-      cy.get('#workflow-actions-menu-button').click();
-      cy.get('#workflow-actions-menu >> [data-testid="reset-button"]').click();
-      cy.get('[data-testid="workflow-reset-type-select"]').click();
-      cy.get(
-        '#reset-type-select-options > li[role="option"]:nth-child(2)',
-      ).click();
-      cy.get('#reset-reason').type('test');
-      cy.get(confirmBtn).click();
-
-      cy.wait('@reset-workflow-api');
-      cy.waitForWorkflowAPIs();
-
-      cy.get('[data-testid="workflow-reset-alert"]').should('be.visible');
-      cy.get(confirmBtn).should('not.be.visible');
-    });
-
     it('to an arbitrary workflow task if the workflow is running, and the action is enabled', () => {
       cy.get('#workflow-actions-menu-button').click();
       cy.get('#workflow-actions-menu >> [data-testid="reset-button"]').click();
-      cy.get('[data-testid="workflow-reset-type-select"]').click();
-      cy.get(
-        '#reset-type-select-options > li[role="option"]:nth-child(3)',
-      ).click();
+      cy.get('#reset-event-5').click();
```

Comment:
Not a blocker, but how do you feel about this value being hard-coded?

Reply:
Yeah it's not ideal, but it's based on `event.id` our JSON mocks, so I think it's tolerable. I can still use a less brittle selector here though, so I'll change it 👍